README.md

```diff
@@ -2,3 +2,21 @@ ROS 2 examples
 ==============
 
 To see some of these examples in use, visit the ROS 2 Tutorials page: https://index.ros.org/doc/ros2/Tutorials/
+
+Branches
+--------
+
+This repo targets different ROS distributions on different branches.
+You must use the branch matching the ROS distro being used when building from source.
+
+Ony [non-EOL ROS distributions](https://index.ros.org/doc/ros2/Releases/) are supported.
```

Comment:
just a thought, but it might be a good idea to quickly elaborate/explain what "are supported" means here, especially since the table below lists both supported and unsupported distros.

also, `Ony` -> `Only`

Reply:
Actually, instead of a negative ("non-EOL"), I'd suggest the positive ("currently supported").  So maybe rewrite the sentence something like:

```suggestion
Changes are only accepted for [currently supported ROS distributions](https://index.ros.org/doc/ros2/Releases/).
```